.github/workflows/windows_unit_tests.yml

```diff
@@ -58,6 +58,11 @@ jobs:
           conda config --add channels conda-forge
           conda activate curr_py
           conda install numba -q -y
+      - name: Install sktime
+        run: |
+          . $env:USERPROFILE\Miniconda3\shell\condabin\conda-hook.ps1
+          conda activate curr_py
+          pip install sktime
```

Comment:
@ParthivNaresh is this still necessary? What happens if we delete it and sktime is pip-installed at the same time as the other packages? If `sktime` must be installed before `pmdarima`, or after it, we can put that ordering into our `requirements.txt`, right?